### PR TITLE
CVE-2011-1183

### PR DESCRIPTION
**Pull request by montemoranon**
Updated the .yml file for CVE-2011-1183

### CODE REVIEW

File: cves/CVE-2011-1183.yml [hunk 1]
```diff
@@ -10,27 +10,27 @@ CWE_instructions: |
   Please go to cwe.mitre.org and find the most specific, appropriate CWE entry
   that describes your vulnerability. (Tip: this may not be a good one to start
   with - spend time understanding this vulnerability before making your choice!)
-CWE:
+CWE: CWE-269 - Improper Privilege Management
```
Comment (alex-mechler) [inline cves/CVE-2011-1183.yml]: This should be just the CWE number based on what we went over in class.

File: cves/CVE-2011-1183.yml [hunk 2]
```diff
@@ -43,7 +43,13 @@ description_instructions: |
...
   expressions" is fine, and security phrases like "invalid write" are fine to
   keep too.
-description:
+description: |
+  The basic configurations for a Tomcat web application are set in the web.xml file. 
+  This file includes a config called "login-config" which sets a level of security 
+  for the application by whitelisting url's based on a user's role. This bug occurred 
+  when this configuration was not set and a configuration called "metadata-complete" 
+  was set to true, unintentionally allowing users to "bypass intended access restrictions" 
+  through HTTP requests.
```
Comment (alex-mechler) [inline cves/CVE-2011-1183.yml:52]: What access restrictions could be bypassed by exploiting this?

File: cves/CVE-2011-1183.yml [hunk 3]
```diff
@@ -116,7 +124,9 @@ discovered:
     If there is no evidence as to how this vulnerability was found, then you
     may leave the entries blank except for "answer", BUT please write down
     where you looked in "answer".
-  answer:
+  answer: | 
+    After searching the git log for the bug ID and following each of the links on cve.mitre.org, 
+    there was no evidence found of the vulerability discovery
```
Comment (alex-mechler) [inline cves/CVE-2011-1183.yml]: This just feels like some odd wording. While it works, I feel that it could be made clearer by wording it in another way.

Comment (alex-mechler) [reply cves/CVE-2011-1183.yml]: Maybe something along the lines of `"There was no information found about how the vulnerability was discovered in the git logs and on cve.mitre.org."`

File: cves/CVE-2011-1183.yml [hunk 4]
```diff
@@ -139,7 +149,10 @@ interesting_commits:
     emerging themes?
     If there are no interesting commits, demonstrate that you completed this
     section by explaining what happened between the VCCs and the fix.
-  answer:
+  answer: |
+    The thing with this issue is that it was introduced via an 'incorrect fix' to two
```
Comment (alex-mechler) [inline cves/CVE-2011-1183.yml]: I would say to get rid of `"The thing with this issue is that"` part. It feels very informal and in my opinion detracts from the overall statement.

Comment (montemoranon) [reply cves/CVE-2011-1183.yml]: Will do this, thanks.

File: cves/CVE-2011-1183.yml (hunk 1 above)
Comment (montemoranon) [reply cves/CVE-2011-1183.yml]: I may have missed the class when we talked about this. Were we restricted to a subset of the common weaknesses?

File: cves/CVE-2011-1183.yml (hunk 2 above)
Comment (montemoranon) [reply cves/CVE-2011-1183.yml:52]: It was difficult to find information on the specifics of this (probably for security reasons). I will look into finding more details if there are any out there.

File: cves/CVE-2011-1183.yml (hunk 3 above)
Comment (montemoranon) [reply cves/CVE-2011-1183.yml]: Agreed.

File: cves/CVE-2011-1183.yml (hunk 1 above)
Comment (montemoranon) [reply cves/CVE-2011-1183.yml]: I changed it to just the number, if that's what you meant.

Comment (alex-mechler) [reply cves/CVE-2011-1183.yml]: Yup, just the number. We talked about it the Friday it was due.

File: cves/CVE-2011-1183.yml [hunk 5]
```diff
@@ -10,27 +10,27 @@ CWE_instructions: |
...
   set to true.
-curated: false
+curated: true
 
 reported_instructions: |
   Was there a date that this vulnerability was reported to the team? You can
   find this in changelogs, blogs, bug reports, or perhaps the CVE data.
   Please enter your date in YYYY-MM-DD format.
-reported:
+reported: 2011-03-17
```
Comment (LeonKuhne) [inline cves/CVE-2011-1183.yml:27]: Apaches website says, relating to CVE-2011-1183, "This was identified by the Tomcat security team on 13 April 2011 and made public on 17 May 2011."

**Comment by LeonKuhne** [comment]
+5 virtual points

**Pull request merged**